[amazonechocontrol] IllegalStateException on command PUSH_AUDIO_PLAYER_STATE

Here's the log: 

```
2025-11-27 17:18:12.528 [DEBUG] [trol.internal.handler.AccountHandler] - Processing CommandTO{command='PUSH_AUDIO_PLAYER_STATE', payload='{"dopplerId":{"deviceSerialNumber":"G090RF04822602QH","deviceType":"A7WXQPH584YP"},"audioPlayerState":"INTERRUPTED","quality":{"name":"High Definition","badge":{"altText":"https://music-provider-logos.s3.amazonaws.com/badges/AmazonMusic/HD.png"},"stats":{"codec":"flac","dataRateInBitsPerSecond":1039704,"samplingRateInHertz":44100}},"error":false,"errorMessage":null,"mediaReferenceId":"cf01d166-b941-4848-b5f3-f2b060b9b589:2","destinationUserId":"A17LO2P97IT6AO"}', timeStamp=1764260292431}
2025-11-27 17:18:12.528 [DEBUG] [control.internal.handler.EchoHandler] - Handle push command PUSH_AUDIO_PLAYER_STATE
2025-11-27 17:18:12.529 [WARN ] [trol.internal.push.PushStreamAdapter] - Exception while processing message
com.google.gson.JsonSyntaxException: java.lang.IllegalStateException: Expected a string but was BEGIN_OBJECT at line 1 column 128 path $.quality
See https://github.com/google/gson/blob/main/Troubleshooting.md#unexpected-json-structure
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:523) ~[?:?]
	at com.google.gson.Gson.fromJson(Gson.java:1359) ~[?:?]
	at com.google.gson.Gson.fromJson(Gson.java:1260) ~[?:?]
	at com.google.gson.Gson.fromJson(Gson.java:1170) ~[?:?]
	at com.google.gson.Gson.fromJson(Gson.java:1107) ~[?:?]
	at org.openhab.binding.amazonechocontrol.internal.handler.EchoHandler.handlePushCommand(EchoHandler.java:1038) ~[?:?]
	at org.openhab.binding.amazonechocontrol.internal.handler.AccountHandler.onPushCommandReceived(AccountHandler.java:628) ~[?:?]
	at org.openhab.binding.amazonechocontrol.internal.push.PushConnection.onPushMessageReceived(PushConnection.java:199) ~[?:?]
	at java.util.ArrayList.forEach(ArrayList.java:1596) ~[?:?]
	at org.openhab.binding.amazonechocontrol.internal.push.PushStreamAdapter.onData(PushStreamAdapter.java:105) ~[?:?]
	at org.eclipse.jetty.http2.HTTP2Stream.notifyData(HTTP2Stream.java:658) ~[?:?]
	at org.eclipse.jetty.http2.HTTP2Stream.onData(HTTP2Stream.java:394) ~[?:?]
	at org.eclipse.jetty.http2.HTTP2Stream.process(HTTP2Stream.java:315) ~[?:?]
	at org.eclipse.jetty.http2.HTTP2Session.onData(HTTP2Session.java:287) ~[?:?]
	at org.eclipse.jetty.http2.HTTP2Connection$ParserListener.onData(HTTP2Connection.java:400) ~[?:?]
	at org.eclipse.jetty.http2.parser.BodyParser.notifyData(BodyParser.java:108) ~[?:?]
	at org.eclipse.jetty.http2.parser.DataBodyParser.onData(DataBodyParser.java:150) ~[?:?]
	at org.eclipse.jetty.http2.parser.DataBodyParser.onData(DataBodyParser.java:145) ~[?:?]
	at org.eclipse.jetty.http2.parser.DataBodyParser.parse(DataBodyParser.java:111) ~[?:?]
	at org.eclipse.jetty.http2.parser.Parser.parseBody(Parser.java:222) ~[?:?]
	at org.eclipse.jetty.http2.parser.Parser.parse(Parser.java:151) ~[?:?]
	at org.eclipse.jetty.http2.HTTP2Connection$HTTP2Producer.produce(HTTP2Connection.java:271) ~[?:?]
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produceTask(EatWhatYouKill.java:362) ~[bundleFile:9.4.57.v20241219]
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:186) ~[bundleFile:9.4.57.v20241219]
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:173) ~[bundleFile:9.4.57.v20241219]
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produce(EatWhatYouKill.java:137) ~[bundleFile:9.4.57.v20241219]
	at org.eclipse.jetty.http2.HTTP2Connection.produce(HTTP2Connection.java:193) ~[?:?]
	at org.eclipse.jetty.http2.HTTP2Connection.onFillable(HTTP2Connection.java:148) ~[?:?]
	at org.eclipse.jetty.http2.HTTP2Connection$FillableCallback.succeeded(HTTP2Connection.java:371) ~[?:?]
	at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:105) ~[?:?]
	at org.eclipse.jetty.io.ssl.SslConnection$DecryptedEndPoint.onFillable(SslConnection.java:555) ~[?:?]
	at org.eclipse.jetty.io.ssl.SslConnection.onFillable(SslConnection.java:410) ~[?:?]
	at org.eclipse.jetty.io.ssl.SslConnection$2.succeeded(SslConnection.java:164) ~[?:?]
	at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:105) ~[?:?]
	at org.eclipse.jetty.io.ChannelEndPoint$1.run(ChannelEndPoint.java:104) ~[?:?]
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.runTask(EatWhatYouKill.java:338) ~[bundleFile:9.4.57.v20241219]
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:315) ~[bundleFile:9.4.57.v20241219]
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:173) ~[bundleFile:9.4.57.v20241219]
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:131) ~[bundleFile:9.4.57.v20241219]
	at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:409) ~[bundleFile:9.4.57.v20241219]
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883) ~[bundleFile:9.4.57.v20241219]
	at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034) ~[bundleFile:9.4.57.v20241219]
	at java.lang.Thread.run(Thread.java:1583) [?:?]
Caused by: java.lang.IllegalStateException: Expected a string but was BEGIN_OBJECT at line 1 column 128 path $.quality
See https://github.com/google/gson/blob/main/Troubleshooting.md#unexpected-json-structure
	at com.google.gson.stream.JsonReader.unexpectedTokenError(JsonReader.java:1831) ~[?:?]
	at com.google.gson.stream.JsonReader.nextString(JsonReader.java:974) ~[?:?]
	at com.google.gson.internal.bind.TypeAdapters$15.read(TypeAdapters.java:461) ~[?:?]
	at com.google.gson.internal.bind.TypeAdapters$15.read(TypeAdapters.java:449) ~[?:?]
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$2.readIntoField(ReflectiveTypeAdapterFactory.java:271) ~[?:?]
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$FieldReflectionAdapter.readField(ReflectiveTypeAdapterFactory.java:561) ~[?:?]
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:519) ~[?:?]
	... 42 more

```

--- a/bundles/org.openhab.binding.amazonechocontrol/src/main/java/org/openhab/binding/amazonechocontrol/internal/dto/push/PushAudioPlayerStateTO.java
+++ b/bundles/org.openhab.binding.amazonechocontrol/src/main/java/org/openhab/binding/amazonechocontrol/internal/dto/push/PushAudioPlayerStateTO.java
@@ -21,7 +21,6 @@ import org.eclipse.jdt.annotation.NonNull;
  */
 public class PushAudioPlayerStateTO extends PushDeviceTO {
     public String mediaReferenceId;
-    public String quality;
     public boolean error;
     public AudioPlayerState audioPlayerState;
     public String errorMessage;

